packages/dds/tree/src/simple-tree/treeApi.ts

```diff
@@ -53,23 +53,29 @@ export interface TreeNodeApi {
 	 * Return the node under which this node resides in the tree (or undefined if this is a root node of the tree).
 	 */
 	parent(node: TreeNode): TreeNode | undefined;
+
 	/**
 	 * The key of the given node under its parent.
 	 * @remarks
 	 * If `node` is an element in a {@link (TreeArrayNode:interface)}, this returns the index of `node` in the array node (a `number`).
 	 * Otherwise, this returns the key of the field that it is under (a `string`).
 	 */
 	key(node: TreeNode): string | number;
+
 	/**
 	 * Register an event listener on the given node.
+	 * @param node - the node who's events should be subscribed to.
+	 * @param eventName - which event to subscribe to.
+	 * @param listener - Callback to trigger for the event. The tree can be read during the callback, but it is invalid to modify the tree during this callback.
```

Comment:
Nit: For all param docs, we should ensure the text starts with proper sentence casing and structure.
```suggestion
	 * @param node - The node who's events should be subscribed to.
	 * @param eventName - Which event to subscribe to.
	 * @param listener - The callback to trigger for the event. The tree can be read during the callback, but it is invalid to modify the tree during this callback.
```